CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 * **[Fix]** Remove unsecure UUID fallback when UUID generation theorically fails, in reality it never fails.
 * **[Fix]** Check for running in App Center Test will now work when using AndroidX instead of the support library.
 * **[Feature]** Add `AppCenter.isRunningInAppCenterTestCloud` to provide method to check if the application is running in Test Cloud.
+* **[Feature]** Change the default offline persistence from 1 day to infinite.
```

Comment:
Please look at previous versions, this should be in App Center Data section.

Reply:
oh oh! checking...

Reply:
done~